tests/Server/TextDocument/HoverTest.php

```diff
@@ -156,6 +156,21 @@ public function testHoverForConstant()
         ], $reference->range), $result);
     }
 
+    public function testHoverForGlobalConstant()
+    {
+        // print TEST_PROPERTY ? 'true' : 'false';
+        // Get hover for TEST_PROPERTY
+        $reference = $this->getReferenceLocations('TEST_PROPERTY')[0];
```

Comment:
Please name it `TEST_DEFINE_CONSTANT` or something like that, `TEST_PROPERTY` is confusing